Resources/views/CRUD/dashboard__action.html.twig

```diff
@@ -0,0 +1,4 @@
+<a class="btn btn-link btn-flat" href="{{ action.url }}">
+    <i class="fa fa-{{ action.icon }}"></i>
+    {{ action.label|trans({}, action.translation_domain|default('SonataAdminBundle')) }}
+</a>
```

Comment:
Please configure your editor to add newline at end of file, it should make the red pictogram disappear.